docs/featurizers.rst

```diff
@@ -116,6 +116,23 @@ AtomConvFeaturizer
 .. autoclass:: deepchem.feat.NeighborListComplexAtomicCoordinates
   :members:
 
+ChemicalFingerprint
```

Comment:
Can you add a section for "Materials Featurizers" that all of these featurizers sits in and add some more documentation (say a paragraph) explaining how materials featurizers differ from the other featurizers and the types of problems they should be applied on?

Reply:
This looks great, thanks!